app/models/message.rb

```diff
@@ -9,19 +10,24 @@ class Message < ActiveRecord::Base
 
   validates :content, presence: true
   validates :groups, length: { minimum: 1, message: I18n.t('model.message.need_groups') }
-  validate :in_group, unless: :is_admin
-  attr_accessor :is_admin
+  validate :in_group, unless: :by_admin
```

Comment:
Perhaps there should be a validation like this as well on `MessageComment`?
Right now it does not validate that you are part of one of the groups who will see the comment.

(I guess our authentication would handle this anyway, but I think it would be nice to add.